Source/SmallBasic.Editor/Components/Pages/Edit/LibraryBody.cs

```diff
@@ -175,6 +180,11 @@ protected override void ComposeTree(TreeComposer composer)
                                 });
 
                                 composer.Element("name", body: () => composer.Text($"{this.Library.Name}.{this.Method.Name}()"));
+
+                                if (this.Method.IsDeprecated || this.Method.NeedsDesktop)
```

Comment:
Rereading some of your old feedback -- did you want unique error ribbons for "deprecated" and "desktop" so that these two different issues have separate descriptive ribbons? I can easily do that instead if that's what you'd like to see :)

Reply:
@estberg thanks a lot! the new ribbon looks great!
Yeah, that's exactly what I suggested. I think distinction is vital for users.
One is something they won't be able to use in this project, and the other they will be able to if they download the desktop version.

Reply:
How about dark gray you have for deprecated and blue (the arrows bg color) for desktop?

Reply:
How's something like this?

<img width="414" alt="Screen Shot 2019-09-08 at 8 52 22 PM" src="https://user-images.githubusercontent.com/30135598/64498641-03519780-d27b-11e9-8bd7-199fa0a8db80.png">

Once you let me know that it's good, I will squash all the commits of this PR.